tcn_hpl/data/vectorize.py

```diff
@@ -205,3 +212,154 @@ def vectorize_window(
             f_vecs[i] = empty_vec
 
     return np.asarray(f_vecs)
+
+
+def obj_det2d_set_to_feature_by_method_new(
+    label_vec: List[str],
+    xs: List[float],
+    ys: List[float],
+    ws: List[float],
+    hs: List[float],
+    label_confidences: List[float],
+    pose_confidence: float,
+    pose_keypoints: List[Dict],
+    obj_label_to_ind: Dict[str, int],
+    top_k_objects: int = 1,
+):
+    """
+    :param label_vec: List of object labels for each detection (length: # detections)
+    :param xs: List of x values for each detection (length: # detections)
+    :param ys: List of y values for each detection (length: # detections)
+    :param ws: List of width values for each detection (length: # detections)
+    :param hs: List of height values for each detection (length: # detections)
+    :param label_confidences: List of confidence values for each detection (length: # detections)
+    :param pose_keypoints:
```

Comment:
Minor issue: missing `pose_confidence` and there are some extra parameters below that don't show above.